views/scss/qti/_qti-base.scss

```diff
@@ -122,6 +122,15 @@ $mainContainer : '.qti-item';
         &.highlighter-cursor {
             cursor: url("../img/qtiIconsPng/marker.png") 0 32, auto;
         }
+
+        hr {
+            border: none;
+            margin: 12px 0;
+            padding: 0;
+            height: 1px;
+            max-height: 1px;
+            background-color: #8c8c8c;
```

Comment:
When separator is selected, by mouse or with Ctrl+A, there's no style change so it's not clear that it's selected. 
If it's difficult to do something about it, please ignore this, it's not important.

Reply:
I finally managed to solve this problem. It's almost impossible to do it using CSS because the `::selection` pseudo doesn't seem to apply on the hr tag, even with `content: '  '` in `::after`... 
So I've modified the plugin to listen for `selectionchange` and toggle a class on the element if it's inside a selection.